test/cases/parsing/chunks/index.js

```diff
@@ -19,4 +19,11 @@ it("should parse a string in require.ensure", function(done) {
 		require("./file").should.be.eql("ok");
 		done();
 	});
-});
\ No newline at end of file
+});
+
+it("should parse a string in require.ensure with arrow function expression", function(done) {
+	require.ensure("./file", require => {
```

Comment:
Could you also add a test with a single expression array function? They behave a bit different in AST.

``` js
require.ensure("./file", require => (require("./file").should.be.eql("ok"), done()))
```